docs/source/release/index.rst

```diff
@@ -13,6 +13,7 @@ Release Notes
 
 
 * :ref:`v4.0.0 <v4.0.0>`
+* :ref:`v4.1.0 <v4.1.0>`
```

Comment:
I think we need to remove this line too?

Reply:
Good catch, thanks, removed.